beacon_node/network/src/router.rs

```diff
@@ -637,6 +644,17 @@ impl<T: BeaconChainTypes> Router<T> {
         });
     }
 
+    /// Handle a `DataColumnsByRoot` response from the peer.
+    pub fn on_data_columns_by_root_response(
+        &mut self,
+        _peer_id: PeerId,
+        _request_id: RequestId,
+        _data_column_sidecar: Option<Arc<DataColumnSidecar<T::EthSpec>>>,
+    ) {
+        // TODO(das) implement `DataColumnsByRoot` response handling
+        unimplemented!()
```

Comment:
Before we merge this, perhaps we want to just leave a comment here rather than panicing?

Reply:
Updated in https://github.com/sigp/lighthouse/pull/5196/commits/3a967847ece611ee5486b4061a8b13434b383258